runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsOverlay.java

```diff
@@ -74,7 +78,7 @@ public Dimension render(Graphics2D graphics)
 				net.runelite.api.Point minimapLocation = npc.getMinimapLocation();
 				if (minimapLocation != null)
 				{
-					graphics.setColor(Color.yellow);
+					graphics.setColor(configManager.getConfiguration("minimap", "npc", Color.class));
```

Comment:
This is not correct way to do this, but, it is a good start. Check how minimap plugin sets the colors for minimap dots, and reverse that (e.g read from NPC minimap dot color, its pixel 1,1 i think)